client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -69,11 +70,12 @@ const DesignSelector: React.FunctionComponent = () => {
 							} }
 						>
 							<span className="design-selector__image-frame">
-								<img
+								<MshotsImage src={ getDesignImageUrl( design ) } alt="" />
```

Comment:
I think for initial roll-out we should keep `isEnabled( 'gutenboarding/mshot-preview' )` so that we can quickly revert to static method if there are problems.